### ISSUE TITLE
Translation with Babele not working

### ISSUE THREAD

**Issue by Anthior** [Rughalt/D35E#331]
There is a lot of problems with Babele. Here are mine:

1. CommonBuffs names are invisible when I use the translated file. I can't activate them manually or through spells/abilities. Though I can still click on the invisible name to see it properly translated.
![Capture d’écran 2020-12-16 122430](https://user-images.githubusercontent.com/77992861/107162091-2e28a980-69a1-11eb-8d6f-dcabac8bb302.png)

2. Items (armor, weapons magic weaponsand other items) are translated in the compendium, but their name appear in english when added to a character's inventory. The description is translated.
![Capture d’écran 2020-12-16 122430](https://user-images.githubusercontent.com/77992861/107162588-aa70bc00-69a4-11eb-8b6a-13478d7df418.png)

3. Classes and races are translated both in the compendiums AND when added to a character's sheet. So it's fine.

4. Spells are translated in the compendium. The description in the "Detail" tab is different from the description in the "description" tab. It would be nice to have them both being the same. There is no way to translate the "Snippet" and the description in the "Detail" tab myself because they do'nt appear in the extracted file used by Babele.
![Capture d’écran 2020-12-16 122430](https://user-images.githubusercontent.com/77992861/107162467-9d070200-69a3-11eb-9d38-43c244b83d03.png)

5. When creating a potion/scroll/wand from a spell, only the name is translated, saying "Potion of 'TranslatedSpellName' ". But the description is in english.
![Capture d’écran 2020-12-16 122430](https://user-images.githubusercontent.com/77992861/107162493-d3448180-69a3-11eb-856a-0fad4ca440bc.png)

6. Translated spells are not recognized by the class system: they are not added automatically when pressing the "Add all known spells for this level". They can added manually, but then an error message appear saying it has been added despite not being on the class' spell list.
![Capture d’écran 2020-12-16 122430](https://user-images.githubusercontent.com/77992861/107162612-e3a92c00-69a4-11eb-9d1e-06296b569d08.png)

So as you can see there is a lot of different problems with Babele. All of them are different and I can't do anything. My players can't play in english, but when translated the files become totally useless because not recocgnized properly by Babele or by the System.
I know you spend way more time than me on this system, but I've also spent a lot of time to translate everything and I'm so disappointed because I can't even use it properly. Thank you for thinking about a solution for this. Send me a PM on Discord if you need more informations.

**Comment by Rughalt** [comment]
I changed your ticket to add numbers to each bullet point.

**Comment by Rughalt** [comment]
1, 3, 4 and 5 were fixed by fixing translation files

**Comment by Rughalt** [comment]
2 is fixed, will be out in 0.90

**Comment by irbian** [comment]
Regarding 6, I don´t receive the warning, but the auto adding is not working (0.96.4)

**Comment by Rughalt** [comment]
I think 6 should be fixed as of 0.98.2, please make a new ticket if it is not.

**Comment by irbian** [comment]
Regarding 6, I´m receiving this: when I click on "add everything"

It stays waiting on "updating..." and on console we have this:

![image](https://user-images.githubusercontent.com/3071767/148573670-1841cac8-5d9d-4dee-a169-d3f522a7c0f1.png)
